integration-tests/contracts/contract_deployer.go

```diff
@@ -63,6 +63,10 @@ type ContractDeployer interface {
 	DeployVRFConsumerV2(linkAddr string, coordinatorAddr string) (VRFConsumerV2, error)
 	DeployVRFCoordinator(linkAddr string, bhsAddr string) (VRFCoordinator, error)
 	DeployVRFCoordinatorV2(linkAddr string, bhsAddr string, linkEthFeedAddr string) (VRFCoordinatorV2, error)
+	DeployDKG() (DKG, error)
+	DeployVRFCoordinatorV3(beaconPeriodBlocksCount *big.Int, linkAddr string, linkEthFeedAddr string) (VRFCoordinatorV3, error)
```

Comment:
```suggestion
	DeployOCR2VRFCoordinator(beaconPeriodBlocksCount *big.Int, linkAddr string, linkEthFeedAddr string) (VRFCoordinatorV3, error)
```

Reply:
asked a question back on this topic, waiting for reply there

Reply:
renamed